remappings.txt

```diff
@@ -1,5 +1,6 @@
 solmate/=lib/solmate/
 permit2/=lib/permit2/
 forge-std/=lib/forge-std/src/
-@openzeppelin/=node_modules/@openzeppelin
+@openzeppelin/contracts/token/ERC721/=lib/ERC721
+@openzeppelin/contracts-v4=node_modules/@openzeppelin/contracts-v4
```

Comment:
i dont think that will work because v3-periphery INonfungiblePositionManager imports `'@openzeppelin/contracts/token/ERC721/IERC721Metadata.sol';` OZ v4 contracts have this file in a different folder so it can't find it. So @openzeppelin/contracts needs to be the name for the old version

Reply:
ohhhhh i see 🤔 thats annoying